src/wolff_producer.erl

```diff
@@ -384,8 +434,9 @@ send_to_kafka(#{sent_reqs := Sent,
   {FlatBatch, Calls} = get_flat_batch(Items, [], []),
   [_ | _] = FlatBatch, %% assert
   BatchSize = erlang:length(FlatBatch),
```

Comment:
Do we need to keep the batch size?

Reply:
It's used in other metrics, but since we are redefining the metrics to mean number of batches rather than individual messages, I guess we don't need to keep it around.  I'll remove it from the request info map.

Reply:
:heavy_check_mark: 
https://github.com/kafka4beam/wolff/pull/41/commits/9013d4a84988430cf03273be6e611acafd53fe23